hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -153,7 +153,7 @@
   private transient HoodieWriteClient writeClient;
 
   public DeltaSync(HoodieDeltaStreamer.Config cfg, SparkSession sparkSession, SchemaProvider schemaProvider,
-                   TypedProperties props, JavaSparkContext jssc, FileSystem fs, HiveConf hiveConf,
+                   TypedProperties props, JavaSparkContext jssc, FileSystem fs, Configuration hiveConf,
```

Comment:
Let us change the variable name to conf?

Reply:
I keep `hiveConf` here for better readability. There are two other configuration `cfg` `props` here, so I believe `hiveConf` could be more clear for the user to know what this is for. WDYT?

Reply:
but it can be misleading for e.g if user does not even configure hive sync right? I also prefer `conf`

Reply:
This config was only used by hive sync, if the user don't config this it will use the default Hadoop config. I don't have a strong opinion here and ok to go with `conf`.

Reply:
ok...After a closer look, I'd still vote for naming this as `hiveConf`. There are `cfg` `props` in `HoodieDeltaStreamer` and `DeltaSync`. If we name it conf, I personally will get confused while reading the code. This `hiveConf` is specifically for Hive Sync, even it's not `HiveConf` type anymore, but `hiveConf` will still represent what it does? WDYT?

Reply:
I still think its confusing to have it as hiveConf...  esp when you are actually creating ` HiveConf` from this down below..

Reply:
changed to `conf`